src/spyglass/sharing/sharing_kachery.py

```diff
@@ -215,3 +215,38 @@ def download_file(analysis_file_name: str) -> bool:
             raise Exception(f"{analysis_file_name} cannot be downloaded")
 
         return True
+
+
+def share_data_to_kachery(
+    restriction={},
+    table_list=[],
+    zone_name="franklab.default",
```

Comment:
Should be franklab.collaborators (seems correct in the docstring)?

Reply:
What role does the zone play here? And related to that who are the most likely users? If the zone enables people from that zone to access it, then collaborators seems like a worth default? Or perhaps by default it should be both zones?